src/core/components/layout/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Layout } from 'antd'
+import { Route, Switch } from 'react-router'
+
+import { LoginPage, NotFoundPage, PortalPage } from '../../../pages'
+import { Content } from '../../../common/ui-kit'
+import Header from '../header'
+import Footer from '../footer'
+
+function PageLayout() {
+  return (
+    <div>
+      <Layout>
+        <Header />
+        <Content>
+          <Switch>
+            <Route exact path="/login" component={LoginPage} />
```

Comment:
We can export all path in another file example constants.js 
```
export const PATH_URL = {
  LOGIN: '/login',
  HOME: '/',
  NOT_FOUND_PAGE: '/*',
}
```
and in index.js just import constant 
```

const { LOGIN, HOME, NOT_FOUND_PAGE } = PATH_URL

 <Route exact path={LOGIN} component={LoginPage} />
 <Route exact path={HOME} component={PortalPage} />
 <Route path={NOT_FOUND_PAGE} component={NotFoundPage} />
```